docker-compose.yml

```diff
@@ -3,25 +3,39 @@ version: '3'
 services:
   arangodb:
     image: arangodb:3.5.1
+    restart: always
+    ports:
+      - 8529:8529
     environment:
       ARANGO_NO_AUTH: 1
 
   spline:
-    image: absaoss/spline-rest-server
+    image: "absaoss/spline-rest-server:0.5.1"
+    restart: always
     ports:
       - 9090:8080
     command: >
-      bash -c "echo 'Initializing Spline DB...'
-      && curl -O -s https://repo1.maven.org/maven2/za/co/absa/spline/admin/0.5.0/admin-0.5.0.jar
-      && java -jar ./admin-0.5.0.jar db-init arangodb://arangodb/spline -s
-      && catalina.sh run"
+      bash -c "
+        until curl --output /dev/null --silent --get --fail http://172.17.0.1:8529
+        do
+          echo waiting for arango db
+          sleep 5
+        done
+        if [ ! -f ./init_completed ]
```

Comment:
Why do you need a marker file? To save on startup time?

Reply:
nope, you simply get "db already exists" if run twice :( it's not cleaniest, but easiest

Reply:
`-s` option tells `admin-cli` to skip initialization if the db is already initialized. Aren't we doing the same thing twice with the marker file?

Reply:
I'll recheck again but i'm 99% sure that it wasn't working

Reply:
Yep

```
spline_1    | Exception in thread "main" java.lang.IllegalArgumentException: Arango Database spline already exists
spline_1    |   at za.co.absa.spline.persistence.ArangoInit$.$anonfun$deleteDbIfRequested$1(ArangoInit.scala:109)
spline_1    |   at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
spline_1    |   at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
spline_1    |   at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
spline_1    |   at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
spline_1    |   at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
spline_1    |   at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
spline_1    |   at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
spline_1    |   at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
spline_spline_1 exited with code 1

```

Reply:
Aha, it means that `admin-cli` has a bug.

Reply:
Looks like an ArrangoDB driver bug - fileed an issue #696

Reply:
Do you agree to remove the marker file if `-s` option works correctly? I'll place a quick workaround in 0.5.2

Reply:
Sure, that problem is the only reason this marker workaround has been added :)

Reply:
PR created - #698

Reply:
I've reverted 'init_completed' part